src/connector/src/sink/encoder/mod.rs

```diff
@@ -90,6 +90,7 @@ pub enum TimestampHandlingMode {
 #[derive(Clone)]
 pub enum CustomJsonType {
     Doris(HashMap<String, (u8, u8)>),
+    Kafka(String),
```

Comment:
This is not a customer json type. We could have `kafka_connect: Option<struct {root_struct_name: String}>` as a field of `JsonEncoder`.

Similarly, I do not think `CustomJsonType::Doris(HashMap<String, (u8, u8)>)` is the correct design. It should be 3 options `enum DateHandlingMode { Days, String }`, `decimal_paramater_table: HashMap<String, (u8, u8)>` and `preserve_fields_order: true`. But it is out of the scope here and we can keep it untouched.